REQUIREMENTS.md

```diff
@@ -0,0 +1,155 @@
+# IPLD Requirements
+
+This document outlines parts of IPLD that should and should not be changed to
+ensure the success of future improvements (especially type systems).
+
+**Definitions**
+
+* Block: A block is a chunk of an IPLD DAG, encoded in a format. Blocks have CIDs.
+* Patch: A piece of an IPLD DAG. Blocks contain patches.
```

Comment:
Can we find another term for this? Patch is commonly used as a verb for selective edits of a structure. There are even existing threads about adding a "patch" API to the existing dag API's in IPFS.

Reply:
We can probably just use subgraph.